Tests/AuthServiceTests/Validators/CreateUserRequestValidator.cs

```diff
@@ -0,0 +1,130 @@
+﻿using DTO.RestRequests;
+using NUnit.Framework;
+using System;
+using UserService.Validators;
+using static UserService.Validators.ErrorsMessages;
+
+
+namespace AuthenticationServiceTests.Validators
+{
+    class CreateUserRequestValidatorTests
```

Comment:
I think we already have the same validator , but it named incorrectly: emailPasswordValidatorTests